docs/design/schedule_algorithm.md

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Please start this doc with a paragraph like "This doc describes details of the scheduling algorithms that Kubernetes uses. For a higher-level overview of the scheduling process, see [and then point to docs/devel/scheduler.md]"